docs/user_guide/Certificates/Certificates.inc

```diff
@@ -13,6 +13,25 @@ SIMP server and are served to the clients over the puppet protocol.
 
    See :ref:`Certificate Management` for more information.
 
-This section provides guidance on installing official certificates or, as an
-interim measure, generating certificates from the Fake (self-signing)
-Certificate Authority provided by SIMP.
+SIMP uses the ``pupmod-simp-pki`` module to help distribute keypairs.
+The global variable, ``simp_options::pki`` determines what parts of the module are
+included.  It can be overridden at several levels if different hosts or
+applications need to hande certificates differently.
+
+``simp_options::pki`` can have one of three settings:
+
```

Comment:
Numbered items aren't formatted quite right.  Need to indent continuation lines.